codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -1011,6 +1020,9 @@ public void operationComplete(ChannelFuture future) throws Exception {
             private void firstWriteComplete(ChannelFuture future, ChannelPromise promise) {
                 Throwable cause = future.cause();
                 if (cause == null) {
+                    // As we just finished our first write which made the stream-id valid we need to re-evaluate
+                    // the writability of the channel.
+                    writabilityChanged(Http2MultiplexCodec.this.isWritable(stream));
```

Comment:
can we rename `writabilityChanged` to `checkForWritabilityChange` (or something similar)? At this point we are not indicating there is a writability change ... we are just checking if there has been a change.